OpenRA.Mods.Common/Widgets/VideoPlayerWidget.cs

```diff
@@ -149,13 +151,13 @@ public override void Draw()
 			if (Video == null)
 				return;
 
-			if (!stopped && !Paused)
+			if (startTime != null && !Paused)
 			{
 				int nextFrame;
 				if (Video.HasAudio && !Game.Sound.DummyEngine)
 					nextFrame = (int)float2.Lerp(0, Video.FrameCount, Game.Sound.VideoSeekPosition * invLength);
 				else
-					nextFrame = Video.CurrentFrameIndex + 1;
+					nextFrame = (int)float2.Lerp(0, Video.FrameCount, (float)((pauseTime ?? DateTime.Now) - startTime.Value).TotalSeconds * invLength);
```

Comment:
1. optional, cache lastFrame. to avoid re-execution while paused. 
2. perhaps also already add support for playing the video as x times the speed, while at it.

Reply:
i assume playing video frames always has to be based on real world seconds and that i does not relate to game logic ticks and game logic speed.

Reply:
Adding speed playback is problematic, as this also needs to speed up the audio played. I would want to leave that out of this pr to not bloat it by providing an playing-audio-speed-manipulation api to the sound engine.